tests/foreman/ui/test_oscapcontent.py

```diff
@@ -35,6 +35,7 @@ def oscap_content_path(module_target_sat):
     return local_file
 
 
+@pytest.mark.skip_if_open("BZ:2167937, 2133151")
```

Comment:
This wont work! Method `skip_if_open` only accepts a single number, so repeat the marker for every number, as below:
```suggestion
@pytest.mark.skip_if_open("BZ:2167937")
@pytest.mark.skip_if_open("BZ:2133151")
```

Reply:
Repeat this for all below !